binder/environment.yml

```diff
@@ -1,5 +1,21 @@
+# a mybinder.org-ready environment for demoing jupyterlab_execute_time
+# this environment may also be used locally on Linux/MacOS/Windows, e.g.
+#
+#   conda env update --file binder/environment.yml
+#   conda activate jupyterlab_execute_time-demo
+#
+name: jupyterlab_execute_time-demo
+
 channels:
   - conda-forge
+
 dependencies:
-  - jupyterlab>=2
-  - nodejs=12
+  # runtime dependencies
+  - python >=3.8,<3.9.0a0
```

Comment:
Not sure what the python breaking version convention is, but maybe this can be wider ? (<4.0.0 ?)

Reply:
This was generated, but I agree. Fixed.